### ISSUE TITLE
DUA 14593 Skinner - Amend - Add 2018 data & update project staff - HOLD for SOW Amend approval - Docs ready to submit 

### ISSUE THREAD

**Issue by MonicaAdamsFoster** [MonicaAdamsFoster/Monica#9]
Need the ES approval from the SOW amendment before can submit this: 
Add 16-17 MCBS,S&C, 06-16 MBSF: NDI, 11-17 MBSF OCPDC, MDS, ACO
Add C. Chang to DUA
Update E. Meara to Harvard

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/822812349362644/1159070489091235) by [Unito](https://www.unito.io/learn-more)

**Comment by MonicaAdamsFoster** [comment]
Got all docs ready, Stpeh send me the MDS & had saved the NDI. All submitted to ResDAC - original email had wrong DUA #.

**Comment by MonicaAdamsFoster** [comment]
Email in from ResDAC, can't have 2 amendments at once with them. They already have the microsoft, etc. request. They asked if we want to combine this with that or if wew want to wait on this one until that one is done. Responded to Hanna Steph and I would discuss on 12/13 and get back to her. Fwd email to Steph

**Comment by MonicaAdamsFoster** [comment]
Per Steph she emailed Beth to let her know we will put this request on hold until we have the one for the SOW completed.

**Comment by MonicaAdamsFoster** [comment]
On the white board. Will need to update the ES with the one from the amendment we are waiting on.

**Comment by MonicaAdamsFoster** [comment]
Already a Harvard Collab for Chandra, can we just add Ellen to that checklist?

**Comment by MonicaAdamsFoster** [comment]
Per discussion with Steph do not need to change anything on the collab checklist for Harvard. Just update Ellen's Institution information.

**Comment by MonicaAdamsFoster** [comment]
All docs are ready for submission except Ellen's sig addend (and waiting for current amendment to get approved)

**Comment by MonicaAdamsFoster** [comment]
Ellen's doc is ready. Hold for now
